bin/run-dev

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
+pushd $(git rev-parse --show-toplevel)
+
+set -e
 
 # run bin/pull-image
```

Comment:
Suggestion (non-blocking): it is clear now the comment is no longer needed.

Reply:
duh! i can delete these 😁